sympy/geometry/parabola.py

```diff
@@ -96,7 +96,7 @@ def ambient_dimension(self):
         2
 
         """
-        return S(2)
+        return 2
```

Comment:
I'm not sure about this. Should this be a symbolic return value?

Reply:
At the time I just went for a feeling that it is similar to `len`. However, checking now, all other `ambient_dimension` methods return `int` typically `len` of something.

For example:
https://github.com/sympy/sympy/blob/d168c74babaf92ed523cf503e7398a30dc9a235d/sympy/geometry/line.py#L138
https://github.com/sympy/sympy/blob/9dd704b071e5134a0e8beeeb1a2d2599536f5144/sympy/geometry/ellipse.py#L185

So at least it is consistent now. Correct? I do not know.